README.md

```diff
@@ -1,2 +1,90 @@
-# persist-tools
-Ballerina persist tools
+Ballerina Persist Tools
+===================
+
+[![Build](https://github.com/ballerina-platform/persist-tools/actions/workflows/build-timestamped-main.yml/badge.svg)](https://github.com/ballerina-platform/persist-tools/actions/workflows/build-timestamped-main.yml)
+[![Trivy](https://github.com/ballerina-platform/persist-tools/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/persist-tools/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/persist-tools.svg)](https://github.com/ballerina-platform/persist-tools/commits/main)
+[![codecov](https://codecov.io/gh/ballerina-platform/persist-tools/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/persist-tools)
+
+The Persist Tool supports database operations on Ballerina Persistent Layer on top of Ballerina SQL modules without having to write any SQL statements.
+
+## Issues and projects
+
+Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+This repository only contains the source code for the package.
+
+## Build from the source
+
+### Set up the prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+    * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+    * [OpenJDK](https://adoptium.net/)
+
+      > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+2. Export Github Personal access token with read package permissions as follows,
+
+              export packageUser=<Username>
+              export packagePAT=<Personal access token>
+
+### Build the source
+
+Execute the commands below to build from source.
+
+1. To build the package:
+    ```    
+    ./gradlew clean build
+    ```
+2. To run the tests:
+    ```
+    ./gradlew clean test
+    ```
+
+3. To run a group of tests
+    ```
+    ./gradlew clean test -Pgroups=<test_group_names>
+    ```
+
+4. To build the without the tests:
+    ```
+    ./gradlew clean build -x test
+    ```
+
+5. To debug package implementation:
+    ```
+    ./gradlew clean build -Pdebug=<port>
+    ```
+
+6. To debug with Ballerina language:
+    ```
+    ./gradlew clean build -PbalJavaDebug=<port>
+    ```
+
+7. Publish the generated artifacts to the local Ballerina central repository:
+    ```
+    ./gradlew clean build -PpublishToLocalCentral=true
+    ```
+
+8. Publish the generated artifacts to the Ballerina central repository:
+    ```
+    ./gradlew clean build -PpublishToCentral=true
+    ```
```

Comment:
we don't need these two. currently, tools are not supported via central, should pack with the distribution.

Reply:
Updated in [418b6f7](https://github.com/ballerina-platform/persist-tools/pull/12/commits/418b6f743e1545c6ebad9c1a4704f743901efffb)